tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "declaration": true,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "jsx": "react",
+    "lib": ["dom", "es6", "es2019"],
+    "isolatedModules": true,
+    "module": "ES6",
+    "moduleResolution": "node",
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitAny": false,
```

Comment:
[clarification] Does `noImplicitAny=false` make it possible to use Paragon imports (e.g., components) that don't have types defined (in the package or from `@types/paragon`)?

Presumably, this is the config option that lets consuming applications treat TypeScript as opt-in versus requiring everything to be strictly-typed, correct?

Reply:
That's my understanding.  Imported components that don't include type definitions are treated as 'any', and with `noImplicitAny: true` this would generate errors.

Reply:
Got it, cool. Just wanted to sanity check my understanding of that option, too haha :)